lib/json_elixir.ex

```diff
@@ -1,18 +1,23 @@
 defmodule JsonElixir do
-  @moduledoc """
-  Documentation for `JsonElixir`.
+  alias JsonElixir.Process
+
+  @moduledoc ~S"""
+    JSON Elixir is tool to convert json into HTML
```

Comment:
It's usually recommended to align the text to the left, otherwise, it will add those extra spaces to the docs.

```suggestion
  JSON Elixir is tool to convert json into HTML
```